docs/rebuild_admin.rst

```diff
@@ -0,0 +1,404 @@
+Rebuilding an Admin Workstation USB
+-----------------------------------
+
+In cases where an Admin Workstation stick has been lost or destroyed, and no
+backup exists, it is possible to rebuild one. In order to do so, you'll need
+ 
+ - physical access to the SecureDrop servers
+ - 2 USB sticks:
+
+   - Tails Master USB
+   - 1 replacement Admin Workstation USB (USB3 and 16GB or better recommended)
+
+The process requires experience with the Linux command line and Tails, and 
+can take up to 3 hours. If a backup of the SecureDrop application server is available,
+reinstalling the instance may be simpler. An outline of the steps involved is as 
+follows:
+
+ #. Prepare the USB sticks.
+ #. (Optional) Boot the *Application* and *Monitor Server* in single user mode and  reset
+    the shell admin account password.
+ #. Set up SSH access for the new *Admin Workstation*.
+ #. Retrieve SecureDrop configuration settings from the  *Application* and *Monitor Server*.
+ #. Back up and configure the SecureDrop application.
+ #. Run the ``install`` and ``tailsconfig`` commands from the new *Admin Workstation*.
+ #. Complete post-rebuild tasks.
+
+
+Step 1: prepare USB sticks
+==========================
+
+First, create a Tails Master USB using the installation procedure described here: 
+https://tails.boum.org/install/index.en.html
```

Comment:
Right now, you're pointing the user to the Tails download instructions, then to the SecureDrop documentation, which also includes those same download instructions -- including the use of a master to create additional Tails drives. 

How about only asking the user to perform the steps described in
https://docs.securedrop.org/en/release-0.13.0/set_up_tails.html#install-tails
to create a new *Admin Workstation* drive with a persistent volume?

Reply:
done.